README.md

```diff
@@ -9,16 +9,21 @@
   * [Table of Contents](#table-of-contents)
   * [Requirements](#requirements)
   * [Usage](#usage)
+  * [Adding the Nomad Pack Registry](#adding-the-nomad-pack-registry)
+  * [Running a Nomad Pack](#running-a-nomad-pack)
+    * [Testing a Nomad Pack](#testing-a-nomad-pack)
   * [Notes](#notes)
   * [Author Information](#author-information)
   * [License](#license)
 <!-- TOC -->
 
 ## Requirements
 
-* HashiCorp Nomad `1.5.5` or [newer](https://developer.hashicorp.com/nomad/downloads).
-* HashiCorp Nomad Pack `0.0.1` or [newer](https://releases.hashicorp.com/nomad-pack/).
-* `terraform-docs` `0.16.0` or [newer](https://terraform-docs.io/user-guide/installation/).
+- HashiCorp Nomad `1.5.6` or [newer](https://developer.hashicorp.com/nomad/downloads)
+- HashiCorp Nomad Pack `0.0.1` or [newer](https://releases.hashicorp.com/nomad-pack/)
+- a check-out of [@workloads/tooling](https://github.com/workloads/tooling)
+- `terraform-docs` `0.16.0` or [newer](https://terraform-docs.io/user-guide/installation/)
+- `newman` `5.3.2` or [newer](https://learning.postman.com/docs/collections/using-newman-cli/installing-running-newman/)
```

Comment:
It might be worth to put those as "optional", for testing/docs (at face value, not knowing what newman is, it sounds weird having to install some random NPM package)

Reply:
Good call out! Ive added language to this effect